api/src/catalog/catalog.utils.ts

```diff
@@ -9,14 +9,72 @@ import { request } from 'graphql-request';
 
 import winston from '../logging/winston';
 
+/**
+ * Type for Catalog data
+ */
+
+export type SeasonsType = {
```

Comment:
I would recommend removing the `Type` suffix. You wouldn't say `seasonValue` either.

Reply:
so just `Seasons` and `Catalog`?

Reply:
right